tests/suites/constraints/constraints.sh

```diff
@@ -26,5 +29,21 @@ test_constraints_common() {
 			run "run_constraints_vm"
 			;;
 		esac
+
+		destroy_controller "test-constraints"
 	)
 }
+
+# test_constraints_model is concerned with
+test_constraints_model() {
```

Comment:
Move this to the constraints_model.sh file

Reply:
That isn't really how the rest of this test structure is done. Seems like we would breaking the convention set.